### PR TITLE
Fi fixes instalation

### PR DESCRIPTION
**Pull request by fede-pe**
This PR includes:

- Fixes in IMOD installation. Now installing for all Linux distributions (previously only ubuntu)

### CODE REVIEW

**Review by pconesa — APPROVED** [review]
Should we bump the version for a release?

**Comment by fede-pe** [comment]
I think we should wait until jj_fix_continue_ctf_est branch is merged, but yes we should release it, changes are significant.

**Pull request merged**
